app/templates/components/public/session-item.hbs

```diff
@@ -0,0 +1,52 @@
+{{#ui-accordion}}
+  <div class="title">
+    <div class="ui">
+      <h3 class="ui header">
+        {{session.title}}
+        <div class="sub header">
+          {{session.sessionType.name}}
+        </div>
+      </h3>
+      <div class="ui row">Talk</div>
+    </div>
+    <br>
+    <div class="ui grid">
+      <div class="column twelve wide">
+        <div class="session-speakers">
+          {{#each session.speakers as |speaker|}}
+            <img alt="speaker" class="ui mini avatar image" src="{{if speaker.photo.iconImageUrl speaker.image '/images/placeholders/avatar.png'}}">
+          {{/each}}
+        </div>
+      </div>
+      <div class="ui right aligned column four wide" style="padding-left: 0">
+        <div class="">{{session.microlocation.name}} <i class="marker icon"></i></div>
+        <div class="">{{moment-format session.startAt 'hh:mm a / DD-MM-YYYY'}} <i class="wait icon"></i></div>
+      </div>
+    </div>
+    <div class="ui row">
+      <div class="session-description">
```

Comment:
There should be a column inside the row. So put this inside a div with class `column`